server/routes.go

```diff
@@ -797,6 +805,54 @@ func (s *Server) CreateBlobHandler(c *gin.Context) {
 	c.Status(http.StatusCreated)
 }
 
+func (s *Server) IsLocal(c *gin.Context) bool {
```

Comment:
`IsLocal()` probably isn't the right name for this function where it is checking auth, maybe `IsServerKeyAuthorization()`